docs/l4-l7.rst

```diff
@@ -479,3 +515,38 @@ Dynamic Endpoint Discovery functionality is provided by F5ACIServiceCenter and h
 
 ------
+
+**Q.Why older partitions that no longer exist on the BIG-IP are still retrieved by the AS3 endpoint?**
+
+The L4-L7 Application Inventory tab in FASC Application displays a list of all FAST and AS3 applications created on the BIG-IP device as well as on the FASC application. The Application Inventory tab on FASC always pulls the current AS3 configurations present on the BIG-IP device and it gets updated whenever the tab is clicked or reloaded. However in case of misconfigurations or configuration failures on the BIG-IP, users might see certain FAST/AS3 applications displaying on the Application Inventory tab which no longer exist on the BIG-IP.  If you run into this condition,  you can do a GET request through Postman on the following AS3 Endpoint API to verify if the concerned partition is still present on the BIG-IP in the API response.
+
+API URL: https://<BIGIP-IP:Port-Here>/mgmt/shared/appsvcs/declare?showHash=true
+Request type: GET
+Authentication type: Basic Auth (Enter username & password of the BIG-IP here)
+
+Hence, If the concerned partition is present in the API response, please report this issue/bug with the F5 BIG-IP AS3 team.  If the concerned partition is not present in the API response, please report this issue with F5 FASC team at https://github.com/F5Networks/f5-aci-servicecenter/issues
+
+------
+
+**Q.Why do I receive an error while updating the Route domain of an AS3/FAST Application?**
+
+The FASC application throws the following error message while trying to update the Route Domain from one Route Domain to another Route Domain - Error Message - “On updating the Route domain of this application, the Virtual Server of this application would reference the new Route domain. However, the Pool members of this Endpoint group may not reference the new Route domain. To continue, kindly delete this application and create a new one with the required Route domain.” This behavior was observed due to an existing bug in the AS3 Service Discovery plugin v3.41, where on moving an existing AS3 application from one Route Domain to another, the Service Discovery nodes are not updated to the new Route Domain. For example, if an application is mapped to a Tenant | Application Profile | EPG from the APIC and you are trying to update the Route Domain of this application from 0 to a non-zero Route Domain then the Virtual Server of this application will get updated with the newly assigned Route Domain. However, the EPG members from the APIC would still reference the older Route Domain inside the application Pool on the BIG-IP. Here is the link for the bug filed with AS3 Team - https://github.com/F5Networks/f5-appsvcs-extension/issues/669 . As a workaround suggested in the error message, you can delete this AS3/FAST application and create a new one with the desired Route Domain. 
```

Comment:
not sure if " - Error Message - " is part of the actual error message, if not then probably:
"...one Route Domain to another Route Domain: "On updating...
if it is, then still a colon:  "...one Route Domain to another Route Domain: - Error Message - "On updating....